staging/src/k8s.io/apiextensions-apiserver/Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/apiextensions-apiserver",
-	"GoVersion": "go1.8",
+	"GoVersion": "go1.9",
```

Comment:
Is this correct?

Reply:
Did we support 1.9 with kube 1.7? You probably have to switch the go version locally.

Reply:
> You probably have to switch the go version locally.

Yeah, thought so. Will do.